packages/dm-core-plugins/src/view_selector/TabsPlugin.tsx

```diff
@@ -26,7 +26,7 @@ export const TabsPlugin = (
     throw new Error(JSON.stringify(error, null, 2))
   }
   if (isLoading || !viewSelectorItems.length || !selectedViewId) {
-    return <Loading />
```

Comment:
Why remove the loading animation?

Reply:
To avoid too many loading signs pop-up/flickering. And just make the plugins responsible for showing the loading sign.